build.gradle

```diff
@@ -158,7 +158,7 @@ dependencies {
 
     compile 'com.github.tomtung:latex2unicode_2.12:0.2.2'
 
-    errorproneJavac 'com.google.errorprone:javac:1.8.0-beta-132'
+    errorproneJavac 'com.google.errorprone:javac:1.8.0-u20'
```

Comment:
'Why do you change this? 1.8.0-132 is the latest avaiable one for jdk8

Reply:
I think `u-20` is newer. It was released on month after `beta-132`.

Reply:
Ah okay, confusing